### PR TITLE
fix(curriculum): update test 14 Travel Agency Lab

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Test 14 is always passing. I have updated the test.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md [hunk 1]
```diff
@@ -121,8 +121,8 @@ You should have a `p` element below your first `h2` element.
 
 ```js
 assert.strictEqual(
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md:123]: It works but it throws. If you want you can do the following:

```js
const h2 = document.querySelector('h2');
assert.exists(h2);
assert.strictEqual(
  h2?.nextElementSibling,
  document.querySelectorAll('p')?.[1]
);
```

File: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md [hunk 2]
```diff
@@ -121,8 +121,8 @@ You should have a `p` element below your first `h2` element.
...
-  document.querySelector('h2')?.nextElementSibling,
-  document.querySelectorAll('p')?.[1]
+  document.querySelector('h2')?.nextElementSibling.tagName,
+  document.querySelectorAll('p')?.[1].tagName
```
Comment (moT01) [inline curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md:125]:
```suggestion
  document.querySelector('h2')?.nextElementSibling.tagName, 'P'
```

Wouldn't something like this be better? Same with the test three before this one (I can't suggest changes there on GH).

**Comment by zairahira** [comment]
I went with Dario's solution.

File: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md (hunk 2 above)
Comment (Dario-DC) [reply curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md]: Yeah, this makes more sense. And maybe we should change the other test too @zairahira

Comment (zairahira) [reply curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md]: @Dario-DC,
I was able to find one more similar test and I have updated it.

File: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md [hunk 3]
```diff
@@ -120,9 +119,10 @@ assert.equal(document.querySelectorAll('h2')[0]?.innerText, 'Packages');
 You should have a `p` element below your first `h2` element.
 
 ```js
+const h2 = document.querySelector('h2');
+assert.exists(h2);
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md:123]: With these changes you don't need these two lines anymore
```suggestion
```

**Review by Dario-DC — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md (hunk 3 above)
Comment (zairahira) [reply curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md]: You're right. I've removed them.

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Pull request merged**